SEGGER/SEGGER_RTT.c

```diff
@@ -167,6 +167,26 @@ Additional information:
   #endif
 #endif
 
+#ifndef   SEGGER_RTT_MEMCMP

```

Comment:
is this a new segger version or your modifications?
I'm afraid that those changes may get lost if someone updates with new version of segger_rtt.

Reply:
It's my modifications. I was assuming that Segger people follow this repo and approve these changes so that they become they also take them.
If not, what do you suggest? Contact them through their official website?

Reply:
I'm not sure how to push it to segger.

Reply:
> Segger is not part of the zephyrproject-rtos Git-Repo.
> However, we will review the changes and extend the official RTT sources if we
> come to the conclusion that it is a good addition to the current RTT state.
> I will keep you posted.

I got this reply after opening a ticket in Segger support.

Reply:
After opening a ticket in support.segger.com, they proposed the update V7.86h that gives equivalent changes and improved initialization. I tried to leave the `SEGGER_RTT.c` untouched and follow their suggestion to not call at all `SEGGER_RTT_Init()` in the chained application, but, even if working, I saw that the APIs using use `INIT()` aren't used in Zephyr log backend. So I added a change only in `SEGGER_RTT_Init()`, using the configuration I'm aware of that distinguish the cases when we want to to a forced init of CB or just check if it's already done.